explorer/site-config/hca-dcp/dev/Index/common/constants.ts

```diff
@@ -0,0 +1,13 @@
+// App dependencies
+import { SUMMARY } from "app/components/Index/common/entities";
+
+/**
+ * Summary display order.
+ */
+export const SUMMARIES = [
+  SUMMARY.ESTIMATED_CELLS,
+  SUMMARY.SPECIMENS,
+  SUMMARY.DONORS,
+  SUMMARY.FILES,
+  SUMMARY.FILE_SIZE,
```

Comment:
My apologies F, we are no longer displaying file size (see [corresponding functionality ticket](https://github.com/clevercanary/data-browser/issues/118)). Can you remove file size from HCA and LungMAP please?

Reply:
You can also remove any file size-related functionality (e.g. constants, transformers).